src/cryptonote_core/service_node_list.cpp

```diff
@@ -1665,23 +1665,26 @@ void service_node_list::state_t::process_new_ethereum_tx(
         log::info(
                 globallogcat,
                 fg(fmt::terminal_color::green),
-                "New service node {} tx from ethereum: {} (THIS NODE) @ height: {}; awaiting "
+                "New service node {} tx ({}) from ethereum: {} (THIS NODE) @ height: {}; awaiting "
                 "confirmations",
                 type,
+                cryptonote::get_transaction_hash(tx),
                 snpk,
                 block_height);
     else if (tx.type == cryptonote::txtype::ethereum_staking_requirement_updated)
         log::info(
                 globallogcat,
-                "Service node staking requirement tx from ethereum changing to {} SENT @ height: "
+                "Service node staking requirement tx ({}) from ethereum changing to {} SENT @ height: "
                 "{}; awaiting confirmations",
                 cryptonote::print_money(val),
+                cryptonote::get_transaction_hash(tx),
                 block_height);
     else
-        log::info(
+        log::debug(
```

Comment:
Do you really want to change all of these to `debug`?  Note that these (unlike the "THIS NODE" ones above) apply to `logcat` rather than `globallogcat` and so won't show up at the default oxend log level.

Reply:
I've reverted this, I misunderstood how the logs were working thinking that this would be printed on all nodes at the default log settings.